bin/npm-cli.js

```diff
@@ -78,6 +78,7 @@
       const pkg = require('../package.json')
       let notifier = require('update-notifier')({pkg})
       if (
+        npm.config.get('update-notifier'),
```

Comment:
Actually, I'd like this to be further up so you don't even require the notifier package at all?

Reply:
sure, that makes sense. how about combined w/ the conditional on line 77?